pettingzoo/butterfly/knights_archers_zombies/src/players.py

```diff
@@ -44,7 +55,7 @@ def update(self, action):
                 self.rect.x += math.cos(move_angle) * self.speed
                 self.rect.y -= math.sin(move_angle) * self.speed
             elif action == 2 and self.rect.y < const.SCREEN_HEIGHT - 40:
-                self.rect.x += math.cos(move_angle) * self.speed
+                self.rect.x -= math.cos(move_angle) * self.speed
```

Comment:
Was this an old bug? Or what's the reason for this change?

Reply:
Yep, this was an old bug, the agent couldn't move backwards when it was facing 90 or 270 degrees.